### ISSUE TITLE
keybase.pub Use case: as off-site host

### ISSUE THREAD

**Issue by techmagus** [keybase/kbfs#1008]
Hi,

  I am curious if it is within keybase.pub's "terms" or "design" to be used an off-site hosting of sort?

  These are for files like: .css; .eot; .woff; .woff2; .svg; maybe even .gif; .jpg; .png

  I recently tested it actually, .css works fine but the font files are getting a CORS error (if I understood it correctly, the font extensions are not allowed in keybase.pub's CORS setting to be accessed by other domains).

  Basically, is it allowable? If yes, then the font files not being allowed in CORS was by-design I believe?

  Thank you.
